src/app/component/user/components/habit/all-habits/components/habits-list-view/habits-list-view.component.scss

```diff
@@ -0,0 +1,5 @@
+div {
```

Comment:
why element `div` but not some class?

Reply:
it was just mock styles, this styles will be part of work of another developer. I removed it.